packages/dicom/typescript/src/read-image-dicom-file-series.ts

```diff
@@ -34,14 +34,14 @@ async function readImageDicomFileSeries(
     throw new Error('"input-images" option must have a length > 1')
   }
 
-  options.inputImages.forEach(async (value) => {
+  await Promise.all(options.inputImages.map(async (value) => {
```

Comment:
Cool!  

Should we just return the transformed image instead of pushing into array?

`const inputs = await Promise.all(`

Reply:
good idea, but `inputs` can also have content from other input's in the general case.

Reply:
ahh, I always make this mistake. For future, would it make sense to have some file naming convention to indicate generated files? like `*.gen.ts, *.gen.js` etc?